docs/sections/contributor_guide/developer_setup.rst

```diff
@@ -2,4 +2,173 @@
+|                     | ``coverage``                                               |
++---------------------+------------------------------------------------------------+
+| ``make test``       | Equivalent to                                              |
+|                     | ``make lint && make typecheck && make unittest``, plus     |
+|                     | checks defined CLI scripts                                 |
++---------------------+------------------------------------------------------------+
+
+
+Note that ``make format`` is never run automatically, to avoid
+reformatting under-development code in a way that might surprise the
+developer. A useful development idiom is to periodically run
+``make format && make test`` to perform a full code-quality sweep
+through the code.
+
+The order of the targets above is intentionally, and possibly useful:
+``make format`` will complain about certain kinds of syntax errors that
+would cause all the remaining code-quality tools to fail (and may change
+line numbers reported by other tools, if it ran after them);
+``make lint`` provides a good first check for obvious errors and
+anti-patterns in the code; ``make typecheck`` offers a more nuanced look
+at interfaces between functions, methods, etc. and may spot issues that
+would cause ``make unittest`` to fail.
+
+The ``make test`` command is also automatically executed when ``conda``
+builds a ``uwtools`` package, so it is important to periodically run
+these tests during development and, crucially, before merging changes,
+to ensure that the tests will pass when CI builds the ``workflow-tools``
+code.
+
+In addition to the ``make devshell`` command, two other ``make`` targets
+are available for use *outside* a development shell, i.e. from the base
+conda environment (requires presence of the ``condev`` package):
+
+
++------------------+-------------------------------------------------------+
+| Command          | Description                                           |
++==================+=======================================================+
+| ``make env``     | Creates a conda environment based on the ``uwtools``  |
+|                  | code                                                  |
++------------------+-------------------------------------------------------+
+| ``make meta``    | Update ``recipe/meta.json`` from ``recipe/meta.yaml`` |
++------------------+-------------------------------------------------------+
+| ``make package`` | Builds a ``uwtools`` conda package                    |
++------------------+-------------------------------------------------------+
+
+
+These targets work from the code in its current state in the clone.
+``make env`` calls ``make package`` automatically to create a local
+package, then builds an environment based on the package.
+
+Building condev locally
+-----------------------
+
+As an alternative to installing a prebuilt package from
+`anaconda.org`_, the ``condev`` package can be
```

Comment:
https://anaconda.org/maddenp/condev might be a better target for the "anaconda.org" link here, as it will take the user directly to the page for `condev` itself. Something like

> As an alternative to installing [a prebuilt package from anaconda.org](https://anaconda.org/maddenp/condev) ...